build/import/Packages.targets

```diff
@@ -44,34 +44,34 @@
 
     <!-- VS SDK -->
     <PackageReference Update="Microsoft.VSSDK.BuildTools"                                             Version="17.0.63-dev17-g3f11f5ab" />
-    <PackageReference Update="Microsoft.VisualStudio.ComponentModelHost"                              Version="17.0.32-g9d6b2c6f26"/>
+    <PackageReference Update="Microsoft.VisualStudio.ComponentModelHost"                              Version="17.0.67-g9e37b637e6"/>
     <PackageReference Update="Microsoft.VisualStudio.Composition"                                     Version="16.9.20"/>
     <PackageReference Update="Microsoft.VisualStudio.Data.Core"                                       Version="9.0.21023" />
     <PackageReference Update="Microsoft.VisualStudio.DataDesign.Common"                               Version="17.0.0-preview-2-31223-026"/>
     <PackageReference Update="Microsoft.VisualStudio.Data.Services"                                   Version="9.0.21023" />
     <PackageReference Update="Microsoft.VisualStudio.DataTools.Interop"                               Version="17.0.0-preview-2-31223-026" />
     <PackageReference Update="Microsoft.VisualStudio.Designer.Interfaces"                             Version="17.0.0-preview-2-31223-026" />
-    <PackageReference Update="Microsoft.VisualStudio.ImageCatalog"                                    Version="17.0.0-preview-2-31223-026" />
+    <PackageReference Update="Microsoft.VisualStudio.ImageCatalog"                                    Version="17.0.0-previews-1-31321-016" />
```

Comment:
strange that this is now versioned as "previews"

Reply:
Yes, not sure why they switched from preview-2 to previews-1 again but this is a higher version than what was there before